keps/3122-expose-flavors-in-localqueue-status/README.md

```diff
@@ -0,0 +1,141 @@
+# KEP-3122: Expose Flavors in LocalQueue Status
+
+<!-- toc -->
+- [Summary](#summary)
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposal](#proposal)
+  - [User Stories (Optional)](#user-stories-optional)
+    - [Story 1](#story-1)
+  - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+  - [Risks and Mitigations](#risks-and-mitigations)
+- [Design Details](#design-details)
+  - [Test Plan](#test-plan)
+      - [Prerequisite testing updates](#prerequisite-testing-updates)
+    - [Unit Tests](#unit-tests)
+    - [Integration tests](#integration-tests)
+  - [Graduation Criteria](#graduation-criteria)
+- [Implementation History](#implementation-history)
+- [Drawbacks](#drawbacks)
+- [Alternatives](#alternatives)
+<!-- /toc -->
+
+## Summary
+
+This KEP introduces a new status field in LocalQueue, allowing users to see
+all currently available ResourceFlavors in the LocalQueue.
+
+## Motivation
+
+Currently, users without RBAC access to ResourceFlavors cannot view the list
+of available flavors. Depending on the RBAC rules, users might also lack read
+access to ClusterQueues. Providing users with information about the available
+flavors is useful, as it gives them an idea of the capabilities provided by
+a LocalQueue (e.g., a flavor might include newer GPUs).
+
+### Goals
+
+- Provide a possibility to see all currently available ResourceFlavors in 
+  the LocalQueue.
+
+### Non-Goals
+
+- Verify that the ResourceFlavors exist and show only the existing flavors.
```

Comment:
It might be a side question, but what happens when a non-existing RF is referenced? Is the CQ and / or LQ marked as Active=False with some message indicating the problem?

Reply:
We already discussed with @varshaprasad96 this case [here](https://github.com/kubernetes-sigs/kueue/pull/3143#discussion_r1777949483). Just to be in context.

Reply:
Thanks, I missed that discussion. @varshaprasad96 wdyt about shortening the name and making the model more flexible?

Reply:
Having the resource name specified would be a big +1 for us! We were depending on the cluster admin to document the resources belonging to a particular RF, but this would make it way more easier if it sounds reasonable to include them in the status.